Misc/NEWS.d/next/Library/2018-08-16-16-47-15.bpo-20849.YWJECC.rst

```diff
@@ -0,0 +1,6 @@
+shutil.copytree now accepts a new exist_ok keyword argument. Pass True to
+allow passing a destination directory that already exists. Any existing
+files in an existing destination tree will be overwritten by matching
+files from the source tree. Defaults to False to preserve backwards
+compatibility with the previous behavior of raising an exception when the
+destination directory already exists. Patch by Josh Bronson.
```

Comment:
I think that this level of details belong in the reST docs and possibly in the docstring, but news entries are typically shorter.

Reply:
Shortened the news entry in 9a1016b5b0.